staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -72,6 +72,9 @@ type KubeSchedulerConfiguration struct {
 	// then scheduler stops finding further feasible nodes once it finds 150 feasible ones.
 	// When the value is 0, default percentage (5%--50% based on the size of the cluster) of the
 	// nodes will be scored.
+	// Note: This field will be overridden by the profile-level PercentageOfNodesToScore when that is not zero.
+	// DEPRECATED: Please use the profile-level PercentageOfNodesToScore to configure this instead.
+	// TODO(#95446): Remove PercentageOfNodesToScore from KubeSchedulerConfiguration once v1beta1 is removed.
```

Comment:
This will still be valid in v1beta1, so I don't think we should say "deprecated" here.